webrender/src/renderer.rs

```diff
@@ -4013,6 +4020,47 @@ impl Renderer {
         }
         self.device.end_frame();
     }
+
+    // Sets the blend mode. Blend is unconditionally set if the "show overdraw" debugging mode is
+    // enabled.
+    fn set_blend(&self, mut blend: bool) {
+        if !self.debug_flags.contains(DebugFlags::SHOW_OVERDRAW) {
```

Comment:
shouldn't this be the other way around? `if self.debug_flags.contains(DebugFlags::SHOW_OVERDRAW) { blend = true; }`

Reply:
Doh!